_gettingstarted/3-python-docx.md

```diff
@@ -267,4 +267,12 @@ Python-docx can be used for a variety of use cases and projects, making it a ver
 
 - **Word Document Conversion:** Create a script to convert Word documents to other formats, such as PDF, HTML, or plain text. This can be useful for archiving, sharing, or publishing documents in different formats.
 
-In this course, we have explored the versatile and user-friendly Python-docx library, which empowers users to interact with Microsoft Word documents effortlessly. From creating new documents and adding text, images, and tables to formatting content and saving the final result, learners have acquired a comprehensive set of skills to tackle a wide range of document processing tasks. By understanding Python-docx's capabilities, learners can now confidently embark on projects such as automated report generation, resume builders, and educational tools, streamlining document management and unleashing the potential of this powerful library. Armed with this newfound knowledge, learners can enhance productivity, save valuable time, and explore exciting opportunities in their document-centric endeavors.
\ No newline at end of file
+In this course, we have explored the versatile and user-friendly Python-docx library, which empowers users to interact with Microsoft Word documents effortlessly. From creating new documents and adding text, images, and tables to formatting content and saving the final result, learners have acquired a comprehensive set of skills to tackle a wide range of document processing tasks. By understanding Python-docx's capabilities, learners can now confidently embark on projects such as automated report generation, resume builders, and educational tools, streamlining document management and unleashing the potential of this powerful library. Armed with this newfound knowledge, learners can enhance productivity, save valuable time, and explore exciting opportunities in their document-centric endeavors.
+
+## Useful Links
+* [python-docx 0.8.11 documentation](https://python-docx.readthedocs.io/en/latest/)
+* [PyPi - python-docx](https://pypi.org/project/python-docx/)
```

Comment:
Here we can mention our own package URL also right because we also show the description that's why